src/app/loans/loans-view/loan-account-actions/disburse/disburse.component.ts

```diff
@@ -54,7 +54,7 @@ export class DisburseComponent implements OnInit {
    * and initialize with the required values
    */
   ngOnInit() {
-    this.maxDate = this.settingsService.businessDate;
+    this.maxDate = this.settingsService.maxFutureDate;
```

Comment:
Would you please move these changes into a new PR? It has nothing to do with the "charge payment method"?